index.js

```diff
@@ -12,8 +17,15 @@ function init() {
 
 	const selectedText = editor.getSelectedText();
 	const text = selectedText || editor.getText();
+	const config = atom.config.get('perfectionist');
 
-	postcss(perfectionist(atom.config.get('perfectionist'))).process(text, {
+	if (editor.getGrammar().scopeName === 'source.css.scss') {
```

Comment:
Per original issue, it should be the inverse. SCSS unless there's a `.css` extension:

> This is how I did SCSS support in atom-autoprefixer: https://github.com/sindresorhus/atom-autoprefixer/blob/99bac7cf2f44a41dc552143441240e4c4a4cf9b5/index.js#L17-L22 **Pretty much just use the SCSS parser unless the file is .css. That makes it work for SCSS on unsaved buffers too.**